Packages/com.unity.inputsystem/Documentation~/TimingOptimizeForFixedUpdate.md

```diff
@@ -0,0 +1,112 @@
+# Optimize for fixed-timestep or physics-based scenarios
```

Comment:
I wonder if this section could be divided into two pages somehow - there is a mixture of conceptual and task based information here, which makes it tricky to navigate. I'm thinking the stuff from line 47 could be the task content, and the stuff before would be the conceptual info.

I also wonder whether merging the info from the non-physics based stuff would be a good idea or not. You know the topic better but I wonder whether you could have an introduction to optimizing input events which explains both approaches, and then separate pages on the task content.

I'll leave it to you to decide though!